api/v1alpha3/zz_generated.conversion.go

```diff
@@ -1,3 +1,4 @@
+//go:build !ignore_autogenerated_core_v1alpha3
```

Comment:
@marosset did you use go 1.17 to run make generate? If so can you please run it again with 1.16? The project still expected 1.16

Reply:
Let me doublecheck and update the generated files.

Reply:
I re-ran `make generate` with go 1.16.8 and pushed the changes.